internal/commands/tui.go

```diff
@@ -541,47 +548,13 @@ func Render(items []list.Item, cmds Commands, errors []string) error {
 	l.Styles.Title = titleStyle
 	l.Styles.PaginationStyle = paginationStyle
 	l.Styles.HelpStyle = helpStyle
-	// remove some extra keys from next/prev as used for other things
-	l.KeyMap.NextPage.SetKeys("right", "l", "pgdown")
-	l.KeyMap.PrevPage.SetKeys("left", "h", "pgup")
 	l.Filter = CustomFilter
 
-	l.AdditionalFullHelpKeys = func() []key.Binding {
-		return []key.Binding{
-			key.NewBinding(
-				key.WithKeys("m"),
-				key.WithHelp("m", "toggle read"),
-			),
-			key.NewBinding(
-				key.WithKeys("M"),
-				key.WithHelp("M", "show/hide read"),
-			),
-			key.NewBinding(
-				key.WithKeys("f"),
-				key.WithHelp("f", "toggle favourite"),
-			),
-			key.NewBinding(
-				key.WithKeys("F"),
-				key.WithHelp("F", "show/hide all favourites"),
-			),
-			key.NewBinding(
-				key.WithKeys("r"),
-				key.WithHelp("r", "refresh feed"),
-			),
-		}
-	}
-	l.AdditionalShortHelpKeys = func() []key.Binding {
-		return []key.Binding{
-			key.NewBinding(
-				key.WithKeys("o"),
-				key.WithHelp("o", "open link"),
-			),
-		}
-	}
+	ListKeyMap.SetOverrides(&l)
```

Comment:
I think we will need an `updaetKeybinds()` function that we can call at various points with filter state/list state/num items etc. Set is good for the initial but related to my other comment, the state seems to get confused after transitioning, might be solved by having an update func.